### ISSUE TITLE
Feature - Adding a server

### ISSUE THREAD

**Issue by sk364** [rohitrajwani/programming-club-IIITDM#1]
A server will help the website handle DB queries and routes much better.
This can become a great project to enhance the productivity in Web Development for the club.

**Comment by rohitrajwani** [comment]
Yes, it was actually planned that way, but it depends on the club's office bearers, whether they want such a feature or not..... I guess @Apptica (Saurabh Joshi) being the former coordinator should look at it....

**Comment by Apptica** [comment]
The website is currently completely static so i think it does not requires a server for the time being. After this asutosh and I was making the same in django and that requires a database to run as the website is dynamic. We will be opening that project soon once we get people to contribute. Everybody busy with their own intern stuffs. :)

**Comment by sk364** [comment]
You can open this project in https://github.com/OpenC-IIIT/
